### PR TITLE
added onnx example

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

File: docs/models_repository.md [hunk 1]
```diff
@@ -5,10 +5,13 @@ This guide will help you to create a models repository for serving with the Open
 
 
 ## Creating Repository for Models in **Intermediate Representation (IR)** format
-The AI models to be served with OpenVINO&trade; Model Server should be in Intermediate Representation (IR) format (where the graph is represented in .bin and .xml format). Tensorflow, Caffe and MXNet trained models can be converted using Model_Optimizer from  OpenVINO&trade; toolkit. Follow the steps from  [model optimizer documentation](https://software.intel.com/en-us/articles/OpenVINO-ModelOptimizer) to convert your model.
+The AI models to be served with OpenVINO&trade; Model Server should be in Intermediate Representation (IR) format (where the graph is represented in .bin and .xml format) on in [ONNX](https://onnx.ai/) format.
```
Comment (mzegla) [inline docs/models_repository.md:8]:
```suggestion
The AI models to be served with OpenVINO&trade; Model Server should be in Intermediate Representation (IR) format (where the graph is represented in .bin and .xml format) or in [ONNX](https://onnx.ai/) format. 
```

File: docs/models_repository.md [hunk 2]
```diff
@@ -20,19 +23,22 @@ models/
...
 - Each model should be stored in a dedicated directory (model1 and model2 in the examples above) and should include sub-folders
 representing its versions (1,2, etc). The versions and the sub-folder names should be positive integer values. 
 
-- Every version folder _must_ include a pair of model files with .bin and .xml extensions; however, the file name can be arbitrary.
+- Every version folder _must_ include model files. IR model files must be with .bin and .xml extensions. ONNX model must have .onnx extension. The file name can be arbitrary.
 
-- Each model in IR format defines input and output tensors in the AI graph. By default OpenVINO&trade; model server is using 
-tensors names as the input and output dictionary keys.  The client passes the input values to the gRPC request and 
+- Each model defines input and output tensors in the AI graph. By default OpenVINO&trade; model server is using 
+tensors names as the prediction input and output names.  The client passes the input values to the gRPC request and
```
Comment (mzegla) [inline docs/models_repository.md:41]: "By default OpenVINO&trade; model server is using tensors names as the prediction input and output names"

It's a little bit  confusing at the first sight. Maybe we could reformulate it to be more descriptive?
Also we could drop gRPC here as the same occurs for HTTP requests.

Take a look at below proposal:
"Each model defines input and output tensors in the AI graph. 
The client passes data to model input tensors by filling appropriate entries in the request inputs map.
Prediction results appear in model output tensors when inference is finished and can be read from response outputs map. By default OpenVINO&trade; model server is using model
tensors names as inputs and outputs names in requests and responses. "

Does it make sense?

File: docs/ovms_quickstart.md [hunk 3]
```diff
@@ -130,3 +130,78 @@ python face_detection.py --batch_size 1 --width 600 --height 400 --input_images_
...
+        content = file.read()
+
+    img = np.frombuffer(content, dtype=np.uint8)
+    img = cv2.imdecode(img, cv2.IMREAD_COLOR)  # BGR format
+    # format of data is HWC
+    # add image preprocessing if needed by the model
+    img = cv2.resize(img, (224, 224))
+    img = img.astype('float32')
+    #convert to RGB instead of BGR if required by model
+    #img = img[:, :, [2, 1, 0]]
```
Comment (mzegla) [inline docs/ovms_quickstart.md:177]: Can you remove unused, commented code?

File: docs/models_repository.md [hunk 4]
```diff
@@ -20,20 +23,24 @@ models/
...
 
-- Every version folder _must_ include a pair of model files with .bin and .xml extensions; however, the file name can be arbitrary.
+- Every version folder _must_ include model files. IR model files must be with .bin and .xml extensions. ONNX model must have .onnx extension. The file name can be arbitrary.
 
-- Each model in IR format defines input and output tensors in the AI graph. By default OpenVINO&trade; model server is using 
-tensors names as the input and output dictionary keys.  The client passes the input values to the gRPC request and 
-reads the results by referring to the correspondent tensor names. 
+- Each model defines input and output tensors in the AI graph. The client passes data to model input tensors by filling appropriate entries in the request inputs map.
+  Prediction results can be read from response outputs map. By default OpenVINO™ model server is using model
+  tensors names as inputs and outputs names in prediction requests and responses. The client passes the input values to the gRPC request and
```
Comment (mzegla) [inline docs/models_repository.md:42]: Can we either drop gPRC or mention REST here? Using "gRPC request" leaves me with a question whether for HTTP request is it any different?

**Review by mzegla — APPROVED** [review]

**Review by rasapala — APPROVED** [review]
LGTM

File: docs/ovms_quickstart.md [hunk 5]
```diff
@@ -130,3 +130,78 @@ python face_detection.py --batch_size 1 --width 600 --height 400 --input_images_
 
 In the `results` folder, look for an image that contains the inference results. 
 The result is the modified input image with bounding boxes indicating detected faces.
+
+That concludes the prediction using the model in IR format. It can be repeated for ONNX model like presented below in step 9.
+
+### Step 9: Prediction Use Case with an ONNX model
```
Comment (dkalinowski) [inline docs/ovms_quickstart.md:136]: This step could be moved to other documentation file

**Review by dkalinowski — APPROVED** [review]

**Pull request merged**